gradoop-examples/gradoop-examples-temporal/src/main/java/org/gradoop/examples/common/TemporalCitiBikeGraph.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright © 2014 - 2019 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.examples.common;
+
+import org.gradoop.flink.model.impl.epgm.LogicalGraph;
+import org.gradoop.flink.util.FlinkAsciiGraphLoader;
+import org.gradoop.flink.util.GradoopFlinkConfig;
+import org.gradoop.temporal.model.impl.TemporalGraph;
+import org.gradoop.temporal.model.impl.pojo.TemporalVertex;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+/**
+ * Class to provide an example graph representing a dump from the citibike rentals of NYC.
```

Comment:
This class should at least include a reference to the license of the provided data, maybe even in the header of the file.
See https://www.citibikenyc.com/data-sharing-policy
I am also unsure if this class is in accordance with point 2.(b) of that license, since the Apache license allows for commercial use.
There should also be a note that CITI BIKE is a trademark, which should not be used according to 2.(h)

Note that I am not a lawyer, this comment is no legal advice.